Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1625,6 +1854,22 @@ def run(self):
         defer.returnValue(rc)
 
 
+class ExtractStaticAnalyzerTestResults(ExtractTestResults):
+    name = 'extract-static-analyzer-test-results'
+
+    def getLastBuildStepByName(self, name):
+        for step in reversed(self.build.executedSteps):
+            if name in step.name:
+                return step
+        return None
+
+    def addCustomURLs(self):
+        step = self.getLastBuildStepByName(CompareStaticAnalyzerResults.name)
+        step.addURL("View full static analyzer results", self.resultDirectoryURL() + "scan-build-output/results.html")
```

Comment:
better to store this directory name `scan-build-output` in a variable instead of hard-coding in various places.